.github/workflows/gh-pages.yml

```
@@ -0,0 +1,36 @@
+name: Publish Storybook
+on:
+  push:
+    branches:
+      - develop
+      - EGRC-275
```

```suggestion
      # TODO: This line must be removed prior to merging into `develop`
      - EGRC-275
```

Can we add a comment like this just so that we don't forget?